README.md

```diff
@@ -18,7 +18,7 @@ NNI (Neural Network Intelligence) is a toolkit to help users run automated machi
 The tool dispatches and runs trial jobs generated by tuning algorithms to search the best neural architecture and/or hyper-parameters in different environments like local machine, remote servers and cloud.
 
 
-### **NNI [v0.9](https://github.com/Microsoft/nni/releases) has been released! &nbsp;<a href="#nni-released-reminder"><img width="48" src="docs/img/release_icon.png"></a>**
+### **NNI [v1.0](https://github.com/Microsoft/nni/releases) has been released! &nbsp;<a href="#nni-released-reminder"><img width="48" src="docs/img/release_icon.png"></a>**
```

Comment:
@rabbit008 @suiguoxin - may we add the link to Xinlan's 1.0 summary?

Reply:
suggest to add the link in master if the summary is not fully ready @scarlett2018  @rabbit008

Reply:
pr #1505 ,please help to review

Reply:
@rabbit008 Pls add the link in the pr of the 1.0 summary